website/static/js/pages/project-settings-page.js

```diff
@@ -227,7 +227,13 @@ $(document).ready(function() {
       var unchecked = checkedOnLoad.filter('#selectAddonsForm input:not(:checked)');
 
       if(unchecked.length > 0 || checked.length > 0) {
-        return 'The changes on addon setting are not submitted!';
+          return 'The changes on addon setting are not submitted!';
+      }
+    /* Before closing the page, Check whether changes made to category, title or description are updated or not */
+      if (projectSettingsVM.title() !== projectSettingsVM.titlePlaceholder ||
```

Comment:
The change is simpler than you think =) .

This code needs to be relocated to the `ProjectSettings` constructor in `projectSettings.js`. You can add `$(window).on('beforeunload',function() {// your code here}`  there.